src/main/java/seedu/module/model/task/Workload.java

```diff
@@ -57,11 +57,11 @@ public static boolean isValidWorkload(String workload) {
     public String displayUi() {
         switch(workloadLevel) {
         case 1:
-            return LOW_WORKLOAD;
+            return LOW_WORKLOAD + " workload";
```

Comment:
I'm not very sure why we need appending " workload". Would you mind telling me

Reply:
I chose to add it so that it's more obvious that its referring to workload. Looking at the original task details could be somewhat disorienting